mmv1/templates/terraform/examples/apigee_organization_cloud_full.tf.erb

```diff
@@ -53,7 +53,7 @@ resource "google_apigee_organization" "org" {
   description                          = "Terraform-provisioned Apigee Org."
   project_id                           = data.google_client_config.current.project
   authorized_network                   = google_compute_network.apigee_network.id
-  runtime_database_encryption_key_name = google_kms_key.apigee_key.id
+  runtime_database_encryption_key_name = google_kms_crypto_key.apigee_key.id
```

Comment:
This seems unrelated - just wanted to double-check it was included intentionally?

Reply:
Correct, unrelated, but a typo I found as I was writing the full test for instance. I could move this into its own PR but since it was doc only I though it was safe to include here.

Reply:
I'm fine with keeping it here. :-)